src/paths/aftercore/info.ts

```diff
@@ -49,6 +52,8 @@ export class AftercoreInfo implements PathInfo {
             },
           },
         ]);
+      case "menagerie":
+        return getTasks([MenagerieQuest]);
```

Comment:
This should also have KnobQuest in it, so that the "Knob/King" after dependency works. Similar to the organ goal.